v2.0/releases/index.md

```diff
@@ -8,8 +8,26 @@ Use the version selector at the top-right of this page to view a different relea
 
 ## v2.0.0
 
+Notable Features:
+
+- Most of Calico's components have been ported from python to golang, including:
+  calicoctl, libcalico (as libcalico-go), calico-cni, libnetwork-plugin, most of
+  Felix, and parts of calico/node. The golang port allows for better alignment
+  with many of the projects we rely on (e.g. we now use the native APIs for etcd
+  and Prometheus directly).
+- calicoctl introduces a new
+  [resource-based UX model]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands).
```

Comment:
Might want to add a line about **experimental** support for swappable BGP backend (GoBGP) with `--backend`